mod-orders/schemas/item_summary.json

```diff
@@ -15,11 +15,21 @@
     },
     "poLineReceiptStatus": {
       "description": "Receipt status of the PO line",
-      "type": "string"
+      "type": "string",
```

Comment:
These values are different from those in the po_line schema.
https://github.com/folio-org/acq-models/blob/master/mod-orders-storage/schemas/receipt_status.json 
schema can be reused

Reply:
Changed the code according to the reviewer's remarks.